cdap-api/src/main/java/co/cask/cdap/api/annotation/Batch.java

```diff
@@ -70,7 +70,7 @@
  * </p>
  * 
  * <p>
- * See the <i><a href="http://cask.co/docs/cdap/current/en/">Cask DAP Developer Guides</a></i>
+ * See the <i><a href="http://cask.co/docs/cdap/current/en/developers-manual/index.html">CDAP Developers' Manual</a></i>
```

Comment:
The title is "Developers" plural-possesive: a manual for developers.

I have fixed the URLs. It was the domain that was the problem.